project_tracking/api/project.py

```diff
@@ -463,13 +464,30 @@ def ingest_genpipes(project_id: str):
             return redirect(request.url)
 
         project_id_from_name = db_action.name_to_id("Project", ingest_data[vc.PROJECT_NAME].upper())
-        if project_id != project_id_from_name:
+        if [int(project_id)] != project_id_from_name:
```

Comment:
If project_id  is not castable,  it is goin to crash.

```
 >>> int('fasfqa')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: 'fasfqa'
```
We might whan to mitigate that.

Reply:
But an id being such thing is supposed to break right?